aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -1680,13 +1687,27 @@ TORCH_IMPL_FUNC(scatter_reduce_two)
   // See issue https://github.com/pytorch/pytorch/issues/74770
   TORCH_WARN_ONCE("scatter_reduce() is in beta and the API may change at any time.");
 
+  dim = at::maybe_wrap_dim(dim, self.dim());
+  auto mut_out = const_cast<Tensor&>(out);
+
+  if (!self.is_same(mut_out)) {
+    mut_out.copy_(self);
```

Comment:
there is no need to do this, `out.copy_` works

Reply:
fixed!